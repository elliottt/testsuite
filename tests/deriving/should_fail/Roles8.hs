{-# LANGUAGE RoleAnnotations, GADTs #-}

module Roles8 where

data T1 a@P = K1 a
