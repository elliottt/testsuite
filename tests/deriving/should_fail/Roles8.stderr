
Roles8.hs:5:1:
    Role mismatch on variable a:
      Annotation says Phantom but role Representational is required
    In the data declaration for ‛T1’
